src/coreclr/vm/methodtable.cpp

```diff
@@ -2881,6 +2892,397 @@ void  MethodTable::AssignClassifiedEightByteTypes(SystemVStructRegisterPassingHe
 
 #endif // defined(UNIX_AMD64_ABI_ITF)
 
+#if defined(TARGET_LOONGARCH64)
+int MethodTable::getFieldTypeByHnd(CORINFO_CLASS_HANDLE clh)
```

Comment:
This name is misleading. It doesn't get field type, but rather its encoded size and classification. Maybe `getFieldSizeClassificationByHnd` would be a more appropriate name.

Reply:
Thanks for your guidance~